zeppelin-interpreter/src/main/java/org/apache/zeppelin/conf/ZeppelinConfiguration.java

```diff
@@ -117,37 +118,25 @@ private void loadXMLConfig(@Nullable String filename) throws ConfigurationExcept
     }
   }
 
-  public static ZeppelinConfiguration create() {
-    if (conf != null) {
-      return conf;
-    }
-    return ZeppelinConfiguration.create(null);
+  public static ZeppelinConfiguration load() {
```

Comment:
I see the impl change, but what's the benefit of renaming method `create` to `load`

Reply:
With this name change, it was easier to see the usage. Also, I think `create` as a method name is not really well chosen, it loads the configuration from a file. I already have a `reload` method in mind, but that is far in the future.